resources/lang/en/frontend.php

```diff
@@ -87,6 +87,33 @@
 		'nodownloads' => 'There are no downloads!',
 	],
 	
+	'emails' => [
```

Comment:
Wait I said to leave these as the full strings as you were doing in the JSON file, just put them in PHP